cargo-near-build/src/near/build/mod.rs

```diff
@@ -94,7 +98,9 @@ pub fn run(args: Opts) -> eyre::Result<CompilationArtifact> {
         &builder_version_info,
         &crate_metadata,
         override_cargo_target_path_env,
+        &output_paths,
     )?;
+    env_keys::print_nep330_env();
```

Comment:
nit: I would keep on top as before

Reply:
This one is much closer to section with other env vars (set for abi build), not set externally outside of docker,
so it's better if it's here, if anyone wants to inspect output for any of these values, because he will see all of them in one place.
Wasm build has one more env variable, not listed for abi build